src/category_theory/zigzag_2.lean

```diff
@@ -75,7 +94,12 @@ instance : has_coe Δ_ Δ :=
 instance category_Δ_ : category Δ_ :=
 { hom := λ n m, { f : fin (n+1) → fin (m+1) | monotone f ∧ f 0 = 0 ∧ f (fin.last _) = fin.last _ },
   id := λ n, ⟨id, by obviously⟩,
-  comp := λ l m n f g, ⟨g.val ∘ f.val, sorry⟩ }.
+  comp := λ l m n f g, ⟨g.val ∘ f.val,
+  by obviously,
+  /- These two proofs aren't very good since the naming f_property_right_left is done automatically
```

Comment:
@Michael-Howes, do you see these notifications, even though the PR has been merged already?

In cases like this, I'd strongly recommend using `tidy?` to see what `tidy` wants to do, copying and pasting that proof, and then golfing it a bit by hand.

Reply:
I do get these notifications. I'll take a look into `tidy?`. Thanks